src/Tickets/Emails/Admin/Emails_Tab.php

```diff
@@ -30,6 +31,51 @@ class Emails_Tab {
 	 * @var string
 	 */
 	public static $slug = 'emails';
+	
+	/**
+	 * Holder for template object.
+	 *
+	 * @since TBD
+	 *
+	 * @var null|Tribe_Template
+	 */
+	protected $template;
+
+	/**
+	 * Edit section slug.
+	 * 
+	 * @since TBD
+	 * 
+	 * @var string
+	 */
+	public static $edit_section_slug = 'edit_email';
```

Comment:
Can possibly go with `section` so that we have consistency on the levels Settings > Tab > Section? So that we can replicate that wherever we are going to have sections? WooCommerce does it this way, and there are plans to re-structure the settings (in general) to have these levels (Settings > tab > section)